### PR TITLE
feat: yearn confirm

### PR DESCRIPTION
**Pull request by theoboldfrazier**
## Description

Please describe your changes

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [ ] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

### CODE REVIEW

**Review by reallybeard — APPROVED** [review]

**Review by elmutt — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -67,7 +68,7 @@
   },
   "scripts": {
     "show-linked-packages": "find node_modules node_modules/@* -depth 1 -type l -print",
-    "link-packages": "grep -oE '@shapeshiftoss\/[a-z-]*' package.json | grep -v hdwallet | xargs yarn link",
```
Comment (0xdef1cafe) [inline package.json:70]: change this back please yarn keeps munging it, i need to move this to a script

File: src/context/EarnManagerProvider/providers/yearn/api/api.ts [hunk 2]
```diff
@@ -16,19 +16,52 @@ import {
...
+  TxInput
 } from './yearn-types'
 
 export type ConstructorArgs = {
   adapter: ChainAdapter<ChainTypes.Ethereum>
   providerUrl: string
 }
 
+export type YearnVault = {
+  inception: 12022103
```
Comment (0xdef1cafe) [inline src/context/EarnManagerProvider/providers/yearn/api/api.ts:29]:
```suggestion
  inception: number
```

File: src/context/EarnManagerProvider/providers/yearn/api/api.ts [hunk 3]
```diff
@@ -16,19 +16,52 @@ import {
...
+    net_apy: number
+  }
+  endorsed: boolean
+  version: string
+  decimals: number
+  type: string
+  emergency_shutdown: boolean
+}
+
+const lower = (str: string) => str.toLowerCase()
```
Comment (0xdef1cafe) [inline src/context/EarnManagerProvider/providers/yearn/api/api.ts:58]: kinda nicer to use `lodash/toLower` rather than write our own utility in a provider

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Comment by 0xdef1cafe** [comment]
2 are required changes, 1 is stylistic/preference, otherwise looks good.

why is it that we went with the approach of putting this provider in web rather than a liquifier package in lib?

File: package.json (hunk 1 above)
Comment (theoboldfrazier) [reply package.json:70]: Done.

File: src/context/EarnManagerProvider/providers/yearn/api/api.ts (hunk 3 above)
Comment (theoboldfrazier) [reply src/context/EarnManagerProvider/providers/yearn/api/api.ts:58]: Done.

File: src/context/EarnManagerProvider/providers/yearn/api/api.ts (hunk 2 above)
Comment (theoboldfrazier) [reply src/context/EarnManagerProvider/providers/yearn/api/api.ts]: Done.

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**